conda/recipes/rapids-xgboost/meta.yaml

```
@@ -28,13 +28,14 @@ requirements:
     - python
     - cuda-version ={{ cuda_version }}
   run:
-    - {{ pin_compatible('cuda_version', max_pin='x', min_pin='x') }}
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
     {% if cuda_major == "11" %}
     - cudatoolkit
     {% endif %}
     - nccl {{ nccl_version }}
     - python
-    - xgboost {{ xgboost_version }}{{ major_minor_version }}
+    - xgboost {{ xgboost_version }}
```

Think we need this to ensure we get our flavor of `xgboost` build for CUDA

```suggestion
    - xgboost {{ xgboost_version }} rapidsai_cuda*
```